spec/rubocop/cop/rspec/subject_stub_spec.rb

```diff
@@ -321,4 +321,46 @@
       end
     RUBY
   end
+
+  it 'flags when there are several top level example groups' do
+    expect_offense(<<-RUBY)
+      describe Foo do
```

Comment:
Maybe prefix with `RSpec.` to emphasize that those are top-level example groups?